x/dispensation/keeper/distribution.go

```diff
@@ -20,10 +20,10 @@ func (k Keeper) SetDistribution(ctx sdk.Context, ar types.Distribution) error {
 	return nil
 }
 
-func (k Keeper) GetDistribution(ctx sdk.Context, name string, distributionType types.DistributionType, authorisedRunner string) (*types.Distribution, error) {
```

Comment:
I wouldn't bother to change this (but I wouldn't change the PR either, it's fine either way); both spellings are correct.  (we use the z here in the US and Canada, the rest of the world uses an s)